checkly/resource_maintenance.go

```diff
@@ -0,0 +1,168 @@
+package checkly
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	checkly "github.com/checkly/checkly-go-sdk"
+)
+
+func resourceMaintenanceWindows() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceMaintenanceWindowsCreate,
+		Read:   resourceMaintenanceWindowsRead,
+		Update: resourceMaintenanceWindowsUpdate,
+		Delete: resourceMaintenanceWindowsDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"starts_at": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"ends_at": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"repeat_unit": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"repeat_ends_at": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"created_at": {
```

Comment:
`created_at` and `updated_at` should be hidden from users, we should skip those values

Reply:
Done :)